src/main/grammars/org/nest/Commons.mc4

```diff
@@ -53,13 +53,12 @@ grammar Commons extends de.monticore.types.Types, org.nest.Units {
          | condition:Expr "?" ifTrue:Expr ":" ifNot:Expr
          | FunctionCall
          | BooleanLiteral // true & false;
-         | NESTMLNumericLiteral
+         | NumericLiteral Variable
+	 | NumericLiteral
```

Comment:
This used to be my job some years ago. Good to see someone take over now that I've grown tired of it ;-)